src/praisonai/praisonai/cli/configuration/oauth.py

```
@@ -0,0 +1,307 @@
+
+    from urllib.parse import urlencode
+
+    auth_url = f"{config.authorization_url}?{urlencode(params)}"
+
+    if on_prompt:
+        on_prompt(auth_url, None)
+    if open_browser:
+        try:
+            webbrowser.open(auth_url)
+        except Exception:
+            pass
```

Ruff (0.15.18)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)